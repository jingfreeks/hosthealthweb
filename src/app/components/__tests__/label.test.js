import { render } from "@testing-library/react";
import {Label} from "../ui";

describe("Label Component", () => {
  it("renders Label Component to make snapshots", () => {
    const { container } = render(<Label />);
    expect(container).toMatchSnapshot();
  });
});
