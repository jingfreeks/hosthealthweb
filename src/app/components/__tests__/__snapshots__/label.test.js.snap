// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component renders Label Component to make snapshots 1`] = `
<div>
  <label
    class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
  />
</div>
`;
